internal/api/handlers/requestadapters/schedulers.go

```diff
@@ -372,6 +372,9 @@ func fromApiRoomOccupancyPolicyToEntity(roomOccupancy *api.RoomOccupancy) *autos
 	roomOccupancyParam.ReadyTarget = float64(readyTarget)
 
 	downThreshold := roomOccupancy.GetDownThreshold()
+	if downThreshold == 0 {
```

Comment:
do we have a test to assert if default threshold is assigned if == 0?

Reply:
We have tests that validate that inputs without `DownThreshold` actually produce outputs with `DownThreshold: float64(autoscaling.DefaultDownThreshold)`. The `.GetDownThreshold` will return 0 if the value is not set. If set, but is 0, it will fail on the API validation thus it will never reach this code

Reply:
great! ty